testing/tests/DevExpress.ui.widgets/treeViewParts/initialization.js

```diff
@@ -24,19 +26,21 @@ QUnit.module('Initialization', () => {
 
     ['items', 'dataSource', 'createChildren'].forEach((dataSourceOption) => {
         [false, true].forEach((virtualModeEnabled) => {
-            [0, -1, 1.1, '0', 'aaa', null, undefined].forEach(rootValue => {
-                QUnit.test(`rootValue = ${rootValue}, dataSource: ${dataSourceOption}, virtualModeEnabled: ${virtualModeEnabled}`, function(assert) {
-                    const options = createOptions({
-                        dataSourceOption, virtualModeEnabled, rootValue, testItems: [
-                            { id: 1, text: 'item1', parentId: rootValue },
-                            { id: 2, text: 'item2', parentId: 1 }]
-                    });
+            ['single', 'multiple'].forEach(selectionMode => {
```

Comment:
IMHO it's better apply 'selectionMode' enumerating to the next test as well:

https://github.com/DevExpress/DevExtreme/blob/52abf62283ef38ae982a4d0cc6fdb1673a3da0eb/testing/tests/DevExpress.ui.widgets/treeViewParts/initialization.js#L47

Reply:
Yep, fixed https://github.com/DevExpress/DevExtreme/pull/13797/commits/591a7cc4daec8932bb6460b590fa46a475ea4d6d